test/processImage.js

```diff
@@ -970,7 +974,9 @@ describe('express-processimage', () => {
               __dirname,
               '..',
               'testdata',
-              'turtleCroppedNorthEastGm.jpg'
+              isDarwin
+                ? 'turtleCroppedNorthEastGm-darwin.jpg'
+                : 'turtleCroppedNorthEastGm.jpg'
```

Comment:
Same here, the image comparison says the images are 13.83% different, so we could set an accepted mismatch percentage of 15.